packages/react-instantsearch-hooks-dom/src/ui/SearchBox.tsx

```diff
@@ -2,12 +2,7 @@ import React from 'react';
 
 import { cx } from './lib/cx';
 
-import type {
-  ChangeEvent,
-  FormEvent,
-  RefObject,
-  JSXElementConstructor,
-} from 'react';
+import type { ChangeEvent, FormEvent, RefObject } from 'react';
```

Comment:
Same comment as yersterday for these types, we can probably reference them from `React`.

Reply:
I'd rather do that when there's a lint rule making me, as there isn't a difference